randovania/gui/multiplayer_session_window.py

```diff
@@ -750,13 +753,19 @@ def generate_layout(progress_update: ProgressUpdateCallable):
             except AnyNetworkError:
                 # We're interested in catching generation failures.
                 # Let network errors be handled by who called us, which will be captured by handle_network_errors
+                alert_user(self, self._options)
+                # Alert the person who generates on errors since update_game_tab
```

Comment:
I dont like putting comments *underneath* the statement thats relevant. Would prefer something more akin to this (hopefully this fits into 1 line?)
Also text should say more akin to "Alert the person who generates on errors, since 'update_game_tab' doesn't show generation errors to other clients

Reply:
Had to really wrangle the comment to make it fit pre-commit's constraints

```py
alert_user(
    self, self._options
)  # Alert the user who gens on errors, since 'update_game_tab' doesn't show gen errors to other clients
```

Reply:
Why not just above? I'd put a blank line between the comment block explaining why we ignore network errors and the block explaining the alert